workspace/src/app/views/pages/MappingEditor/ExecutionReport/LinkingExecutionReport.jsx

```diff
@@ -1,57 +1,70 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 import silkStore from "../api/silkStore";
 import ExecutionReport from "./ExecutionReport";
 
 /**
  * Displays a transform task execution report.
  */
 export default class LinkingExecutionReport extends React.Component {
```

Comment:
When you touch an "old" component, please consider refactoring it to Typescript and hooks.